include/albatross/src/models/gp.hpp

```diff
@@ -45,6 +45,8 @@ struct Fit<GPFit<CovarianceRepresentation, FeatureType>> {
   static_assert(has_solve<CovarianceRepresentation, Eigen::MatrixXd>::value,
                 "CovarianceRepresentation must have a solve method");
 
+  using Feature = FeatureType;
```

Comment:
Is this alias used anywhere? I don't see any.

Reply:
Yeah good catch, this isn't used in this change ... but I did find it very useful when debugging.  Sometimes you might call `model.fit(features)` but internally those features will get converted to a different type and it's nice to have some (easyish) visibility into which feature was used.